<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientModifyForm</class>
 <widget class="QWidget" name="PatientModifyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>수정</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>리스트</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QListWidget" name="lwListModify"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>검색</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>이름</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leNameModify"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>성별</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbGenderModify"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>진단</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbDiagnosisModify"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>치료</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbTreatmentModify"/>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btnSearchModify">
               <property name="text">
                <string>조회</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSaveModify">
               <property name="text">
                <string>변경</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>정보</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnFileModify">
            <property name="text">
             <string>파일</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="twInfoModify">
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string>이름</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>나이</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>성별</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>진단</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>치료</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>홍길동</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>남성</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>치아우식증</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>스케일링</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbImageModify">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>소견</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QTextEdit" name="teDoctorNote">
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
